Reply/app/src/main/java/com/example/reply/ui/ReplyApp.kt

```diff
@@ -227,4 +238,35 @@ fun ReplyNavGraph(
             }
         }
     }
-}
\ No newline at end of file
+}
+
+@Composable
+private fun ReplyNavHost(
+    modifier: Modifier = Modifier,
```

Comment:
Nit: modifiers tend to be the first optional param, so in this case you'd push it down to the bottom